src/features/pages/options/accumulator/data.tsx

```diff
@@ -0,0 +1,279 @@
+import { StaticImage } from 'gatsby-plugin-image'
+import Image from 'features/components/atoms/image'
+import TickDuration from 'images/svg/options/tick_duration.svg'
+import { Localize, localize } from 'components/localization'
+import { OptionsFAQDataItem } from 'features/components/templates/options-content/type'
+import Link from 'features/components/atoms/link'
+
+export const accumulatorCards: CardContent[] = [
+    {
+        id: 1,
+        header: <Localize translate_text="_t_1-5% growth rate_t_" />,
+        description: (
+            <Localize translate_text="_t_Choose your growth rate to tailor risk/reward. _t_" />
+        ),
+        icon: <IllustrativeMinimalCapitalIcon width="48" height="48" />,
+        color: 'gray',
+        align: 'start',
+        size: 'md',
+    },
+
+    {
+        id: 2,
+        header: <Localize translate_text="_t_45-230 tick duration_t_" />,
+        description: (
+            <Localize translate_text="_t_Fast-paced action with durations up to 230 ticks._t_" />
+        ),
+        icon: <Image src={TickDuration} width="48" height="48" />,
+        color: 'gray',
+        align: 'start',
+        size: 'md',
+    },
+    {
+        id: 3,
+        header: <Localize translate_text="_t_Controlled Risk_t_" />,
+        description: (
+            <Localize translate_text="_t_Potential loss is limited to the initial amount you decide to trade._t_" />
+        ),
+        icon: <IllustrativePayoutIcon width="48" height="48" />,
+        color: 'gray',
+        align: 'start',
+        size: 'md',
+    },
+    {
+        id: 4,
+        header: <Localize translate_text="_t_Practise with demo_t_" />,
+        description: (
+            <Localize translate_text="_t_Hone your strategies risk-free with a demo account._t_" />
+        ),
+        icon: <IllustrativePaidProgrammeIcon width="48" height="48" />,
+        color: 'gray',
+        align: 'start',
+        size: 'md',
+    },
+    {
+        id: 5,
+        header: <Localize translate_text="_t_24/7 support_t_" />,
+        description: (
+            <Localize translate_text="_t_Get round-the-clock assistance from our Customer Support team._t_" />
+        ),
+        icon: <IllustrativeSupport247Icon width="48" height="48" />,
+        color: 'gray',
+        align: 'start',
+        size: 'md',
+    },
+]
+
+export const accumulatorOptionsVideoCard: CardContent[] = [
+    {
+        id: 1,
+        header: <Localize translate_text="_t_Watch: Accumulator options in action_t_" />,
+        description: (
+            <Localize translate_text="_t_Check out our quick video walkthrough of how to trade accumulators on Deriv Trader._t_" />
+        ),
+        content: (
+            <Link
+                target="_blank"
+                url={{
+                    type: 'non-company',
+                    href: 'https://blog.deriv.com/videos/how-to-trade-accumulator-options-on-deriv-trader/',
+                }}
```

Comment:
Use this link -> https://github.com/binary-com/deriv-com/blob/master/src/features/components/quill/shared-link/index.tsx